### PR TITLE
Release 0.0.10

### PR DESCRIPTION
**Pull request by SylvainCorlay**
- Update to 0.0.10
 - With this PR, `python-rapidjson` does not bundle rapidjson's headers. Instead, it has a build-time dependency on the `rapidjson` package
 - Adding myself as a maintainer

cc @lelit

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,20 +18,20 @@ build:
...
   run:
     - python
 
 test:
   imports:
-    - rapidjson 
+    - rapidjson
 
 about:
   home: https://github.com/kenrobbins/python-rapidjson
```
Comment (lelit) [inline recipe/meta.yaml]: Maybe it would be better to point to the new _python-rapidjson_ organization URL here?

Comment (SylvainCorlay) [reply recipe/meta.yaml]: Good catch. Thanks!
Updating....

File: recipe/meta.yaml [hunk 2]
```diff
@@ -17,23 +18,23 @@ build:
     - vc10  # [win and py34]
     - vc14  # [win and py>34]
   skip: True  # [py<34 or win]
```
Comment (hajapy) [inline recipe/meta.yaml]: The windows build issues with 0.0.8 seem to have been resolved upstream, so we should try to enable it for this release.

Comment (SylvainCorlay) [reply recipe/meta.yaml]: Thanks! So I guess I need to re-add the vc feature for py27 too

Comment (hajapy) [reply recipe/meta.yaml]: I think this package is py3 only: https://github.com/python-rapidjson/python-rapidjson/blob/v0.0.10/setup.py#L21

Comment (SylvainCorlay) [reply recipe/meta.yaml]: gotcha

**Comment by hajapy** [comment]
Looking through the windows build log, it looks like the issue is that `sysconfig.get_config_var('CC')` returns `None` on windows, but setup.py does not expect that possibility https://github.com/python-rapidjson/python-rapidjson/blob/v0.0.10/setup.py#L59

**Comment by lelit** [comment]
I will tweak RJ's setup.py to handle the case.

**Comment by SylvainCorlay** [comment]
Thanks. I can include a patch in the recipe if you don't want to make a release just for that.

**Comment by lelit** [comment]
Well, I was going to release 0.0.11 anyway so... you're welcome! 😃

**Comment by SylvainCorlay** [comment]
Ok. So I will update the PR to 0.11.

**Comment by hajapy** [comment]
Thanks guys! Looks good.

Closes https://github.com/conda-forge/python-rapidjson-feedstock/issues/2

**Pull request merged**